app/models/metric/targets.rb

```diff
@@ -70,14 +70,10 @@ def self.capture_storage_targets(hosts)
     hosts.flat_map(&:storages).uniq.select { |s| Storage.supports?(s.store_type) & s.perf_capture_enabled? }
   end
 
-  def self.capture_vm_targets(targets, parent_class, options)
```

Comment:
hmm, why was options passed into this method in the first place?  Is it to keep the interfaces to these methods similar?  cc @Fryguy